### ISSUE TITLE
Inventory section empty

### ISSUE THREAD

**Issue by computerguy366** [ArnesSI/netbox-inventory#152]
Netbox 3.7.4 running in Docker

Built with the Inventory plugin and the plugin seems to be functional in Netbox, but all of the items under "Asset management" are empty.

What did I miss that would have populated the Inventory plugin with all the devices I currently have in Netbox?

**Comment by alehaa** [comment]
An asset is a physical item separately managed from devices and thus you need to create assets independently (all pages will be empty after installation). However assets can be assigned to devices. 

Think of it this way:
* When you buy a switch, this is an asset. It keeps being an asset as long as you possess this switch. The only information relevant for this asset is when you have bought it, metadata like purchase and delivery and maybe where the switch is stored or used right now. 
* In contrast a device is the representation of hardware actively being used. For a core switch it is not relevant when it was bought or if the hardware was replaced from time to time. However, you might associate an asset that is used as this device right now.

So in short: An asset is the hardware and persists from purchase to garbage, even when stored in a shelf, while a device is just the timespan when you put the device in a rack and use it for a specific purpose.

**Comment by computerguy366** [comment]
Thanks for that!

I guess I was SORT of using Netbox devices as assets already.  We have a ton of devices in netbox that really are Assets.  Seems a now a daunting task to add them a second time and then associate them to the device already in Netbox.

I guess I'll play around with it and see.  Unless I'm still misunderstanding.

**Comment by alehaa** [comment]
Maybe what you'll need is the feature request discussed in issue #18 for migrating devices to assets one time. If you use devices for asset management and these should be deleted in favor of an asset, maybe you can export the devices as CSV and import them as assets again? Have a look on the available CSV import columns.

**Comment by computerguy366** [comment]
I'm looking at the import functions.  Will take some time to check that out.  I'm still working on wrapping my head around the inventory plugin.  I may create a new clean install of netbox with the inventory plugin to figure out how it all goes together as far as the workflow and connections.  Maybe that will help me understand better.

Thank you for your prompt response!!

**Comment by matejv** [comment]
@alehaa thank you for taking the time and providing an excellent explanation.

Just to add a bit more insight into why this plugin came to be. While you can indeed keep track of your inventory of device assets as you already to  @computerguy366 the same is not possible in netbox for modules and inventory items. Since we have plenty of all kinds of SFP modules that need to be tracked and because we could not keep track of our stock in netbox plain, we developed this plugin.
